redux/slices/business.ts

```diff
@@ -35,11 +39,24 @@ export const businessSlice = createSlice({
       newArray[index] = action.payload;
       state.businesses = newArray;
     },
+    deleteBusiness(state, action: PayloadAction<string>) {
+      const index = state.businesses.findIndex(
```

Comment:
this would be a little bit more readable as: 

const newState = state.business.filter(
        (business) => business!.id !== action.payload
 );
state.businesses = newState;